src/App.js

```diff
@@ -1,32 +1,39 @@
-import React, { Component } from "react";
+import React, { Component, useState, useEffect } from "react";
 import Balance from "./components/Balance";
 import CalcPayment from "./components/CalcPayment";
 import Payments from "./components/Payments";
 import currencies from "./data/currencies";
 import "./App.css";
 
-class App extends Component {
```

Comment:
Great, I did replace the other classes but didn't notice this one. 🙌  great work